server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/RemoteDelivery.java

```diff
@@ -125,7 +125,18 @@
  * javax <i>mail.smtp.starttls.enable</i> property. Depending on how strict your security policy is, you might consider
  * <i>mail.smtp.starttls.required</i> as well. Be aware that configuring trust will then be required.<br/>
  * Read <a href="https://javaee.github.io/javamail/docs/api/com/sun/mail/smtp/package-summary.html"><code>com.sun.mail.smtp</code></a>
- * for full information.
+ * for full information.<br/>
+ * <br/>
+ * <b>Mail encoding:</b><br/>
+ * By enabling the javax mail property `mail.smtp.allow8bitmime` to true. James RemoteDelivery tries to convert quoted-printable
```

Comment:
What about making things simpler instead of more documentation?
I mean, I don't expect people to care about javax-mail parameters, we should not even expose the fact we use that library in the first place.
Can we find a solution to always rely on the same javax-mail parameter and prepare the mail before passing it to it?

Reply:
There's a way, we drop the `allow8bitmime` smtp option hardcoded in RemoteDeliveryConfiguration. Then we don't even need to prepare the mail accordingly. If we still keep it. I can not find any solution.

But after all, why need to use the same smtp properties for all mails?

Reply:
If there's no obvious win at using `allow8bitmime` and it's a source of problems, I completely agree to drop its usage.